src/errorRoutes/errors.js

```diff
@@ -3,12 +3,13 @@
  ************/
 
 // Routing
-const express = require("express");
-const app = express();
+import express from "express";
+export const app = express();
 
 
 // Response
-const { InvalidUrl } = require("dotnet-responses");
+import DotnetResponses from "dotnet-responses";
+const { InvalidUrl } = DotnetResponses;
```

Comment:
`dotnet-responses` is a CommonJS package. Thus, we need to destructure it this way.